pymc/tests/test_data.py

```diff
@@ -405,6 +405,18 @@ def test_implicit_coords_dataframe(self):
         assert "columns" in pmodel.coords
         assert pmodel.named_vars_to_dims == {"observations": ("rows", "columns")}
 
+    def test_implicit_coords_xarray(self):
+        xr = pytest.importorskip("xarray")
+        data = xr.DataArray([[1, 2, 3], [4, 5, 6]], dims=("y", "x"))
+        with pm.Model() as pmodel:
+            with pytest.warns(DeprecationWarning):
+                pm.ConstantData("observations", data, dims=("x", "y"), export_index_as_coords=True)
+        assert "x" in pmodel.coords
+        assert "y" in pmodel.coords
+        assert pmodel.named_vars_to_dims == {"observations": ("x", "y")}
+        assert tuple(pmodel.coords["x"]) == (data.coords["x"],)
+        assert tuple(pmodel.coords["y"]) == (data.coords["y"],)
```

Comment:
The two lines above fail because the left hand side are tuples of coordinate values, and the right hand side is a length-1 tuple.